web/src/main/webapp/WEB-INF/jsp/submit/enketoFormServlet.jsp

```diff
@@ -30,6 +30,26 @@
     console.log("Firstr time first:" + firstLoginCheck);
     var currentUser = "currentUser";
     var appName = "RT";
+
+    function isEdge() {
+        var ua = window.navigator.userAgent;
+        var edge = ua.indexOf('Edge/');
+        if (edge > 0) {
+           // Edge (IE 12+) => return version number
+           return parseInt(ua.substring(edge + 5, ua.indexOf('.', edge)), 10);
+        }
+        // other browser
+        return false;
+    }
+
+    // working around for bug favicon on edge(https://jira.openclinica.com/browse/OC-8814)
+    if (isEdge()) {
+        var link = document.querySelector("link[rel*='shortcut icon']") || document.createElement('link');
+        link.type = 'image/x-icon';
+        link.rel = 'shortcut icon';
+        link.href = 'https://www.openclinica.com/wp-content/uploads/favicon192.png';
```

Comment:
@theywa we need to host the pngs inside the app not linked out. Can we change that.

Reply:
@kkrumlian we can but it won't show up on edge. I already change all favicon.ico to favicon.png(on this commit a99bcc25da241faaa049c297bb829f363275b264) but not work. Also tried to change the type to 'image/png' instead of 'image/x-icon' but not work too. So I tried use link like on hosted auth0 login page, since it work.

Reply:
@theywa let's do it. I understand it won't work but I don't want us to pull the file from openclinica.com